lib/typed_ecto_schema/syntax_sugar.ex

```diff
@@ -116,32 +116,51 @@ defmodule TypedEctoSchema.SyntaxSugar do
     end
   end
 
-  defp transform_expression({:timestamps, ctx, []}) do
-    transform_expression({:timestamps, ctx, [[]]})
+  defp transform_expression({:timestamps, ctx, []}, env) do
+    transform_expression({:timestamps, ctx, [[]]}, env)
   end
 
-  defp transform_expression({:::, _, [{function_name, _, [name, ecto_type, opts]}, type]})
+  defp transform_expression({:"::", _, [{function_name, _, [name, ecto_type, opts]}, type]}, env)
```

Comment:
Any reason to go with `:"::"` instead of `:::` (I don't mind either, just curiosity)

Reply:
Elixir 1.13 changed this I believe.

Reply:
(I just applied mix format I believe, using Elixir 1.13)